charts/datadog/templates/_containers-init-linux.yaml

```diff
@@ -14,9 +17,14 @@
 - name: init-config
   image: "{{ include "image-path" (dict "root" .Values "image" .Values.agents.image) }}"
   imagePullPolicy: {{ .Values.agents.image.pullPolicy }}
-  command: ["bash", "-c"]
+  command:
+    - bash
+    - -c
   args:
-    - for script in $(find /etc/cont-init.d/ -type f -name '*.sh' | sort) ; do bash $script ; done
+    - |
+      for script in $(find /etc/cont-init.d/ -type f -name '*.sh' | sort); do
+        bash $script
+      done
```

Comment:
Is this refactor needed for the main purpose of the PR? I'm finding that this particular change is breaking deployments on GKE Autopilot. See https://github.com/DataDog/helm-charts/issues/947